docs/docs/run-node/01-run-node.md

```diff
@@ -149,3 +149,15 @@ In config.toml:
 ```toml
 log_level: "state:info,p2p:info,consensus:info,x/staking:info,x/ibc:info,*error"
 ```
+
+## State Sync
+
+State sync is the act in which a node syncs the latest or close to the latest state of a blockchain. This is useful for users who don't want to sync all the blocks in history. You can read more here: https://docs.cometbft.com/v0.37/core/state-sync
+
+### Local State Sync
+
+Local state sync work similar to normal state sync except that it works off a local snapshot of state instead of one provided via the p2p network. The steps to start local state sync are similar to normal state sync with a few different designs. 
+
+1. As mentioned in https://docs.cometbft.com/v0.37/core/state-sync, one must set a height and hash in the config.toml along with a few rpc servers (the afromentioned link has instructions on how to do this). 
+2. Bootsrapping Comet state in order to start the node after the snapshot has been ingested. This can be done with the bootstrap command `<app> comet bootstrap-state`
+<!-- 3. TODO after https://github.com/cosmos/cosmos-sdk/pull/16060 is merged -->
```

Comment:
Is this blocked on https://github.com/cosmos/cosmos-sdk/pull/16060 then?

Reply:
the cmd behavior itself is standalone and well-defined, but for end user, it usually work together with local snapshot import feature.